### PR TITLE
Add method to get service property

### PR DESCRIPTION
**Pull request by brnv**

### CODE REVIEW

File: dbus/methods.go [hunk 1]
```diff
@@ -199,6 +199,15 @@ func (c *Conn) GetUnitProperty(unit string, propertyName string) (*Property, err
 	return c.getProperty(unit, "org.freedesktop.systemd1.Unit", propertyName)
 }
 
+// GetServiceProperty returns property for given service name and property name
+func (c *Conn) GetServiceProperty(
```
Comment (jonboulle) [inline dbus/methods.go]: does this actually add anything over GetUnitProperty?

Comment (brnv) [inline dbus/methods.go]: Yes

Comment (jonboulle) [inline dbus/methods.go]: ok, can you explain what? :-)

Comment (brnv) [inline dbus/methods.go]: Units and service units are different dbus objects
https://www.freedesktop.org/wiki/Software/systemd/dbus/#serviceunitobjects

Comment (jonboulle) [inline dbus/methods.go]: oops, fair point.

File: dbus/methods.go [hunk 2]
```diff
@@ -199,6 +199,15 @@ func (c *Conn) GetUnitProperty(unit string, propertyName string) (*Property, err
...
+	service string, propertyName string,
+) (*Property, error) {
```
Comment (jonboulle) [inline dbus/methods.go]: to follow convention of the rest of the code, can you unwrap all these lines?

**Comment by jonboulle** [comment]
lgtm after fixing the style

**Comment by brnv** [comment]
Done.

**Comment by lucab** [comment]
Style fixed, all green, merged. Thanks for your contribution! It would be nice to have a test covering it, @brnv do you think you can add it?

**Pull request merged**